ui/app/components/app/transaction-list/transaction-list.component.js

```diff
@@ -100,9 +100,13 @@ export default class TransactionList extends PureComponent {
           )
         }
         <div className="transaction-list__completed-transactions">
-          <div className="transaction-list__header">
-            { t('history') }
-          </div>
+          {
+            pendingLength > 0 && (
```

Comment:
Okay, so the history title is only needed when the "History" tab needs to be divided into pending and history. Makes sense